docs/quickstart.md

```diff
@@ -1,68 +1,334 @@
-## 1. Install Harvest
+Welcome to the NetApp Harvest Getting Started Guide. This tutorial will guide you through the steps required to deploy a basic instance of NetApp Harvest on a Linux platform to monitor an ONTAP cluster.
```

Comment:
In this tutorial, it would probably be better to use systemd service files for all process management. Right now, harvest service files are missing. Section 7 could be changed to Harvest service and the steps included there.

We should add another paragraph under the introduction saying something like this:

This tutorial uses systemd to manage Harvest, Prometheus, and Grafana. If you would rather, run the processes directly, feel free to ignore the sections of the tutorial that setup systemd service files.